README.md

```diff
@@ -1,63 +1,99 @@
 # ManySecured-D3DB
 
-A repo for the ManySecured D3 data management pipeline from claims and behaviours.
+A repo for the management and storage of ManySecured D3 claims.
+
+## Index
+
+1. [D3 Architecture Summary](#d3-architecture-summary)
+2. [ManySecured Working Group Documentation and Specifications](#manysecured-working-group-documentation-and-specifications)
+3. [How to add a new D3 claim](#how-to-add-a-new-d3-claim)
+4. [Project Details](#project-details)
+   1. [Repo Folder Structure](#repo-folder-structure)
+   2. [D3 Claim Workflow](#d3-claim-workflow)
+
+## ManySecured Working Group Documentation and Specifications
 
-Links:
 - [ManySecured Docs](https://specs.manysecured.net/)
 - [ManySecured Docs: D3](https://specs.manysecured.net/d3)
 - [ManySecured Docs: D3 Claims](https://specs.manysecured.net/d3/D3%20claims)
 - [ManySecured Docs: D3 Behaviours](https://specs.manysecured.net/d3/D3%20behaviourss)
 
+## D3 Architecture Summary
+
+![Relationship Graph for the D3 Claim Types](./examples/D3-claim-dep-graph.png)
```

Comment:
Not super important, but what software is this graph made in?
Can we export this as an SVG instead of a PNG?

Reply:
It seems a bit weird `./examples/` might not be the best place to put this too, but I'm not really sure if it's worth making a `./docs/` folder just for a single file.

Maybe it's worth instead adding the image to https://specs.manysecured.net/, then using a link like:

```md
![Relationship Graph for the D3 Claim Types](https://specs.manysecured.net/images/D3-claim-dep-graph.png)
```